AdminUi/apps/admin_ui/lib/core/modals/add_quota_dialog.dart

```diff
@@ -153,7 +153,7 @@ class _AddQuotaDialogState extends State<_AddQuotaDialog> {
   Future<void> _addQuota() async {
     setState(() => _saving = true);
 
-    assert(_selectedMetric != null && _maxAmount != null && _selectedPeriod != null, 'Invalid state');
+    assert(_selectedMetric != null && _maxAmount != null && _selectedPeriod != null, 'Invalid State');
```

Comment:
Missing translation

Reply:
I had a previous review by Timo and he told me to exclude the assert messages from translation, because there is not need to translate them.